include/openPMD/Series.hpp

```diff
@@ -53,34 +53,80 @@ namespace openPMD
 {
 class ReadIterations;
 class WriteIterations;
+class Series;
+class SeriesImpl;
 
-/** @brief  Root level of the openPMD hierarchy.
+namespace internal
+{
+/**
+ * @brief Data members for Series. Pinned at one memory location.
```

Comment:
Note: maybe it's good to describe that `Impl` and `Data` are mixing classes and that `...Internal` is used as the fixed in-memory storage for each interface class.

Reply:
I have added that description to `design.rst`.